tests/files/gce_ubuntu18-flannel-aio.yml

```diff
@@ -5,10 +5,10 @@ cloud_machine_type: "n1-standard-1"
 mode: aio
 
 # Deployment settings
-
+kubeadm_enabled: true
 bootstrap_os: ubuntu
-kube_network_plugin: flannel
-
+kube_network_plugin: flanne
+dynamic_kubelet_configuration: true
```

Comment:
Remove `dynamic_kubelet_configuration: true`
may be have problems here.

Reply:
this job is passing